src/src/tx_main.cpp

```diff
@@ -736,6 +741,15 @@ void loop()
       msp.markPacketReceived();
     }
   }
+
+  /* Send TLM updates to handset if connected + reporting period
+   * is elapsed. This keeps handset happy dispite of the telemetry ratio */
+  if ((connectionState == connected) && (0 != LastTLMpacketRecvMillis) &&
```

Comment:
Can we make this
```(LastTLMpacketRecvMillis != 0)```

It seems a bit confusing writing it as
```value != variable```

Reply:
Sure, there is a reason why usually write like that :)
Compare these cases:
`variable = 0` vs `0 = variable` 
other one gives you the compile time error...

Reply:
Does make sense, just looks really confusing...

Reply:
np, I can edit that... :)

Reply:
np, I can edit that... :)